main.sh

```diff
@@ -29,15 +34,14 @@ if [[ "${GITHUB_REF:?}" != "refs/tags/"* ]]; then
 fi
 tag="${GITHUB_REF#refs/tags/}"
 
-# TODO: Support custom prefix of tags https://github.com/taiki-e/create-gh-release-action/issues/1
-if [[ ! "${tag}" =~ ^v?[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z_0-9\.-]+)?(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
+if [[ ! "${tag}" =~ ^${prefix}v?[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z_0-9\.-]+)?(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
     error "invalid tag format: '${tag}'"
     exit 1
 fi
-if [[ "${tag}" =~ ^v?[0-9\.]+-[a-zA-Z_0-9\.-]+(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
+if [[ "${tag}" =~ ^${prefix}v?[0-9\.]+-[a-zA-Z_0-9\.-]+(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
     prerelease="--prerelease"
 fi
-version="${tag#v}"
+version="${tag#${prefix}v}"
 title="${title/\$tag/${tag}}"
 title="${title/\$version/${version}}"
```

Comment:
It might be nice to support `$prefix` (without the trailing `-`) as one of the [patterns for titles](https://github.com/taiki-e/create-gh-release-action#inputs).
That allows generating a title like `my-crate 1.0.0` from a tag like `my-crate-v1.0.0`.

Reply:
Oh, good idea!